### DISCUSSION TITLE
cpp cross compile

### DISCUSSION THREAD

**Discussion by cecho0** [xmake-io/xmake#4752] [General]
C++cross compilation, the host is an x86_64 Linux system. To compile Linux programs under arm64, the program needs to rely on some third-party libraries, such as poco. How can i write xmake.lua?

**Comment by waruqi** [comment]
open a pr to enable cross platform for poco package in xmake-repo. then use `add_requires("poco")` and `add_packages("poco")`, you can see docs.

**Reply by cecho0** [reply]
If I am using poco in vcpkg, how should I write it?
